src/status_im/common/signals/events.cljs

```diff
@@ -101,6 +101,12 @@
       "waku.backedup.settings"
       (rf/dispatch [:profile/update-setting-from-backup (transforms/js->clj event-js)])
 
+      "waku.backedup.keypair"
+      {:fx [[:dispatch [:wallet/process-keypair-from-backup (transforms/js->clj event-js)]]]}
+
```

Comment:
Is this the signal that is emitted when a new key-pair is synced?

Reply:
No @seanstrom, this is for backed-up data received from Waku when the user recovers his/her profile.

For new key-pair or watch account sync, there is another place we need to add the same event which we can add later given the refactor amount in wallet core events. This PR aims to break down the events for easy dispatching for a single account received from backup or sync.